Main/src/SettingsScreen.cpp

```diff
@@ -343,8 +343,15 @@ class SettingsScreen_Impl : public SettingsScreen
 
 		nk_label(m_nctx, *label, nk_text_alignment::NK_TEXT_LEFT);
 		nk_combobox(m_nctx, displayData.data(), options.size(), &selection, m_buttonheight, m_comboBoxSize);
-		value = options[selection];
-		g_gameConfig.Set(key, value);
+		
+		String newValue = options[selection];
+		if (value != newValue) {
+			value = newValue;
+			g_gameConfig.Set(key, value);
+			// Window cursor
+			Image cursorImg = ImageRes::Create(Path::Absolute("skins/" + value + "/textures/cursor.png"));
```

Comment:
Check that it's actually the skin setting that's being changed? Since this function should be usable with other config keys.

Reply:
My bad didn't saw that it was for any combo menu
I'm thinking maybe should I change the return value of all nk_* embeded element to make them return a boolean about did the value change like ImGui does.

Reply:
That sounds like  it could be useful 👍